FlowCrypt/Models/OrganisationalRule.swift

```diff
@@ -40,7 +40,12 @@ class OrganisationalRules {
 
     /// use when finding out if EKM is in use, to change functionality without actually neededing the EKM
     var isUsingKeyManager: Bool {
-        clientConfiguration.keyManagerUrl != nil
+        // check for empty string
+        guard let urlString = clientConfiguration.keyManagerUrl, urlString.isNotEmpty else {
+            return false
+        }
+        // check is url can be configured
+        return URL(string: urlString) != nil
```

Comment:
Good initiative to add additional checks - thank you! I'd like the to behave differently. If `keyManagerUrl` is not-null and it's either empty or not a proper URL, the app should show an error and log out.

The reasoning is: some admin somewhere meant to set EKM url to something meaningful, but set it to a wrong string. The intention was that the app speaks to some EKM, so we should not default to null (which would make the app change behavior in ways that may not be secure for companies that want to use EKM). The app should just refuse to do things because the Client Configuration is clearly misconfigured - some admin should then go and fix it.

I prefer to surface errors to the users rather then ignore them, if they may have security implications.

Thanks!

Reply:
Added in next commit. Please check `organisational_rules_url_not_valid` error message.